tests/geo_test.go

```diff
@@ -0,0 +1,44 @@
+package tests
+
+import (
+	"testing"
+
+	"github.com/nknorg/tuna"
+)
+
+type testCase struct {
+	f        tuna.IPFilter
+	location tuna.Location
+	result   bool
+}
+
+var testData = []testCase{
+	{
+		f: tuna.IPFilter{
+			Allow: []tuna.Location{{IP: "1.2.3.4"}},
+		},
+		location: tuna.Location{IP: "1.2.3.4"},
+		result:   true,
+	},
+	{
+		f: tuna.IPFilter{
+			Disallow: []tuna.Location{{IP: "1.2.3.4"}},
+		},
+		location: tuna.Location{IP: "1.2.3.4"},
+		result:   false,
+	},
+}
+
+func TestAllowLocation(t *testing.T) {
```

Comment:
`TestAllowLocation` and `TestDisAllowLocation` should be combined into one function, which loop over all testData and compare results